pymoab-py2-18.04

```diff
@@ -0,0 +1,57 @@
+FROM  ubuntu:18.04
+
+# set timezone
+ENV HOME /root
+ENV TZ=America/Chicago
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+ENV HOME /root
+
+RUN apt-get -y --force-yes update
+RUN apt-get install -y --force-yes \
```

Comment:
```suggestion
RUN apt-get -y --force-yes update; \
         apt-get install -y --force-yes \
```

Reply:
What's the reason for moving these to a single RUN command? Any benefit?

Reply:
it make the docker container smaller, because there is less layer of `commit` in it
(each RUN make a step that is stored)

Reply:
Does separating these two make it that much smaller?

Reply:
```Minimize the number of layers
In older versions of Docker, it was important that you minimized the number of layers in your images to ensure they were performant. The following features were added to reduce this limitation:

Only the instructions RUN, COPY, ADD create layers. Other instructions create temporary intermediate images, and do not increase the size of the build.

Where possible, use multi-stage builds, and only copy the artifacts you need into the final image. This allows you to include tools and debug information in your intermediate build stages without increasing the size of the final image.
```

Reply:
from https://docs.docker.com/develop/develop-images/dockerfile_best-practices/